bin/postinstall

```diff
@@ -1,3 +1,42 @@
 #!/usr/bin/env bash
 
-asdf reshim nodejs "$ASDF_INSTALL_VERSION"
+set -e
```

Comment:
We might consider dropping this line.

Reply:
I like being strict when it comes to scripts like this.